packages/destination-actions/src/destinations/dawn-analytics/index.ts

```diff
@@ -0,0 +1,47 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import track from './track'
+
+import identifyUser from './identifyUser'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Dawn Analytics',
+  slug: 'actions-dawn-analytics',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      writeKey: {
+        label: 'Dawn write key',
+        description: 'Your write key from Dawn Analytics.',
```

Comment:
Label: "Your Write Key"
Description: "Log into your account at app.dawnai.com to get your key."